subprojects/packagefiles/yyjson/meson.options

```diff
@@ -0,0 +1,55 @@
+option(
+    'YYJSON_DISABLE_READER',
+    type: 'boolean',
+    description: 'Disable JSON reader',
+    value: false,
+)
+
+option(
+    'YYJSON_DISABLE_WRITER',
+    type: 'boolean',
+    description: 'Disable JSON writer',
+    value: false,
+)
+
+option(
+    'YYJSON_DISABLE_INCR_READER',
+    type: 'boolean',
+    description: 'Disable incremental reader',
+    value: false,
```

Comment:
These should not directly use the names of the upstream defines.  Uppercase is not traditionally used for Meson options and `YYJSON_` is redundant.  Probably `feature` options make sense, in case autodetection is desired in the future - something like:
```suggestion
    'incr-reader',
    type: 'feature',
    description: 'Enable incremental reader',
```

which will default to `auto`.